test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env rspec
+
+require_relative "../../test_helper"
+
+require "yast"
+
+describe "Yast::AutoinstallAskInclude" do
+  module DummyYast
+    class AutoinstallAskClient < Yast::Client
+      def main
+        Yast.import "Profile"
```

Comment:
why do not need it? autoinstall/ask.rb should ensure that it is imported itself. ( yes, quite common problem in our code, but we should improve it )